sentinel/src/transitions/update_scheduled_reports.js

```diff
@@ -1,131 +1,103 @@
+    // also includes changed doc
+    //
+    _getDuplicates: function(doc, callback) {
+        var q = { include_docs: true },
+            view,
+            clinic_id = utils.getClinicID(doc);
 
-        dbPouch.medic.bulkDocs(
-          docs,
-          {
-            all_or_nothing: true,
-            docs: docs,
-          },
-          function(err) {
-            // cancels transition and marks as incomplete
-            if (err) {
-              logger.error('error doing bulk save', err);
-              return resolve();
-            }
-            return resolve(true);
-          }
-        );
-      });
-    });
-  },
-  //
-  // look for duplicate from same year, month/week and reporting unit
-  // also includes changed doc
-  //
-  _getDuplicates: function(doc, callback) {
-    var q = { include_docs: true },
-      view,
-      clinic_id = utils.getClinicID(doc);
+        if (doc.fields.week || doc.fields.week_number) {
+            q.startkey = [doc.form, doc.fields.year, doc.fields.week || doc.fields.week_number, clinic_id];
+            q.endkey = [doc.form, doc.fields.year, doc.fields.week || doc.fields.week_number, clinic_id, {}];
+            view = 'reports_by_form_year_week_clinic_id_reported_date';
+        } else if (doc.fields.month || doc.fields.month_num) {
+            q.startkey = [doc.form, doc.fields.year, doc.fields.month || doc.fields.month_num, clinic_id];
+            q.endkey = [doc.form, doc.fields.year, doc.fields.month || doc.fields.month_num, clinic_id, {}];
+            view = 'reports_by_form_year_month_clinic_id_reported_date';
+        } else {
+            return callback();
+        }
 
-    if (doc.fields.week || doc.fields.week_number) {
-      q.startkey = [
-        doc.form,
-        doc.fields.year,
-        doc.fields.week || doc.fields.week_number,
-        clinic_id,
-      ];
-      q.endkey = [
-        doc.form,
-        doc.fields.year,
-        doc.fields.week || doc.fields.week_number,
-        clinic_id,
-        {},
-      ];
-      view = 'reports_by_form_year_week_clinic_id_reported_date';
-    } else if (doc.fields.month || doc.fields.month_num) {
-      q.startkey = [
-        doc.form,
-        doc.fields.year,
-        doc.fields.month || doc.fields.month_num,
-        clinic_id,
-      ];
-      q.endkey = [
-        doc.form,
-        doc.fields.year,
-        doc.fields.month || doc.fields.month_num,
-        clinic_id,
-        {},
-      ];
-      view = 'reports_by_form_year_month_clinic_id_reported_date';
-    } else {
-      return callback();
+        db.medic.view('medic', view, q, function(err, data) {
+            callback(err, data && data.rows);
+        });
+    },
+    _isFormScheduled: function(doc) {
+        return doc.fields && (
+            doc.fields.month ||
+            doc.fields.month_num ||
+            doc.fields.week ||
+            doc.fields.week_number
+        ) && doc.fields.year;
```

Comment:
The indentation of this file is all wrong. Make sure your IDE is configured correctly then reformat this file.